tests/smoke-tests/within-existing-monorepo/custom-locations.test.ts

```diff
@@ -4,14 +4,10 @@ import path from 'node:path';
 import { afterAll, beforeAll, describe, expect, it } from 'vitest';
 
 import { assertGeneratedCorrectly } from '../../assertions.js';
-import { createAddon, createTmp, install, runScript } from '../../utils.js';
+import { createAddon, createTmp, fixture, install, runScript } from '../../utils.js';
 
 let commonFixtures = {
-  '.prettierrc.js':
-    // prettier-ignore
-    'module.exports = {' + 
-        '  singleQuote: true,' + 
-        '};',
+  '.prettierrc.js': await fixture('.prettierrc.js'),
```

Comment:
by using a file-on-disk for fixturify, we can ensure that all tests using asserting the content of this file have no typos (or the same typos) across all tests.

For this test in particular, the `.prettierrc.js` was missing the `'use strict';` at the top of the file.

Reply:
moved to https://github.com/embroider-build/addon-blueprint/pull/151